src/Logout.tsx

```diff
@@ -23,31 +17,15 @@ const Logout: FC = () => {
   const dispatch = useDispatch()
 
   useEffect(() => {
-    const handleReset = () => {
-      dispatch(reset())
-      dispatch({type: 'USER_LOGGED_OUT'})
-    }
-
     const handleSignOut = async () => {
       if (CLOUD) {
-        /**
-         * We'll need this authSessionCookieOn flag off for tools until
-         * Quartz is integrated into that environment
-         */
-        const url = isFlagEnabled('authSessionCookieOn')
-          ? new URL(`${window.location.origin}${CLOUD_SIGNOUT_PATHNAME}`).href
-          : `${CLOUD_URL}${CLOUD_LOGOUT_PATH}`
-
-        if (isFlagEnabled('authSessionCookieOn')) {
-          handleReset()
-        }
+        const url = new URL(
+          `${window.location.origin}${CLOUD_SIGNOUT_PATHNAME}`
+        ).href
 
         window.location.href = url
         return
       } else {
-        if (isFlagEnabled('authSessionCookieOn')) {
```

Comment:
What was this conditional doing before? Since already `!CLOUD`, is this an OSS flag? I don't see it there, but if it is, I think we need to check with edge before we remove this code.

Reply:
Looks good to me, understanding that this is a `CLOUD`-only flag.